### ISSUE TITLE
JacocoCoverageExtension error on build

### ISSUE THREAD

**Issue by crazy4groovy** [palantir/gradle-jacoco-coverage#14]
Hi, I'm having trouble configuring the plugin w/ Grade 2.11:

```
buildscript {
    repositories {
        jcenter()
        maven {
            url "https://plugins.gradle.org/m2/"
        }

    }
    dependencies {
        classpath 'com.palantir:jacoco-coverage:0.3.0'
    }
}

apply plugin: 'com.palantir.jacoco-coverage'
```

```
$gradlew build

...

A problem occurred evaluating root project 'myproject'.
> Failed to apply plugin [id 'com.palantir.jacoco-coverage']
   > Could not create an instance of type com.palantir.jacoco.JacocoCoverageExtension_Decorated.
```

**Comment by jwierzbo-intel** [comment]
try to add guava dependency:

> buildscript
> 
> {
>    repositories {
>        jcenter()
>   }
>   dependencies {
>        classpath "com.google.guava:guava:18.0"
>    }
> }

**Comment by jtaylor10** [comment]
Ran into the same issue, adding this to the class path fixes it. This should really get captured in the README, it wasn't clear at all until I found this issue.

**Comment by uschi2000** [comment]
should be fixed in 0.4.0
